deployments/dev/environment.py

```diff
@@ -131,6 +131,7 @@ def mksrc(project, snapshot, subgraphs, ma: int = 0):
     mksrc('datarepo-dev-67240cf2', 'hca_dev_f86f1ab41fbb4510ae353ffd752d4dfc__20210901_20210903', 20),
     mksrc('datarepo-dev-e8e0a59a', 'hca_dev_f8aa201c4ff145a4890e840d63459ca2__20210901_20210903', 384),
     mksrc('datarepo-dev-96d8e08c', 'hca_dev_faeedcb0e0464be7b1ad80a3eeabb066__20210831_20210903', 62),
+    mksrc('datarepo-dev-3b058b81', 'hca_dev_dc1a41f69e0942a6959e3be23db6da56__20210827_20220228_dcp14', 10),
```

Comment:
This breaks the sorting and there is already a snapshot for this project in line 121. Please port `mksrc` and `mkdict` from prod2 to this file, including those in https://github.com/DataBiosphere/azul/pull/3898. It will ensure sorting and prevent collisions.